src/universal/components/Dashboard/DashLayout.js

```diff
@@ -0,0 +1,64 @@
+import React, {PropTypes} from 'react';
+import look, {StyleSheet} from 'react-look';
+
+import Helmet from 'react-helmet';
+import {head} from 'universal/utils/clientOptions';
+
+import NotificationBar from 'universal/components/NotificationBar/NotificationBar';
+
+let styles = {};
+
+const DashLayout = (props) => {
```

Comment:
Well this is a kick in the pants: https://github.com/eslint/eslint/issues/3354

If `arrow-parens` is set to always, our async functions fail. Let's go with `as-needed`.